### ISSUE TITLE
ERROR in ./src/router/index.js 39:13 Module parse failed: Unexpected token

### ISSUE THREAD

**Issue by richardhector** [PanJiaChen/vue-element-admin#1654]
Hi all, I'm getting the below issue when running 'npm run build:prod' can anyone helpe`

`ERROR in ./src/router/index.js 39:13
Module parse failed: Unexpected token (39:13)
You may need an appropriate loader to handle this file type.
|     path: '/redirect/:path',
|     component: function component() {
>       return import('@/views/redirect/index');
|     }
|   }]
 @ ./src/main.js 15:0-30 47:10-16

 Build failed with errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! automater-frontent@0.0.1 build: `cross-env NODE_ENV=production env_config=prod node build/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the automater-frontent@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:`

**Comment by Silom** [comment]
It appears the .babelrc configuration only allows development to use import, this might need a fix or a good explanation of why that is perhaps.
Fact is on master if you run npm run build, it will crash.

**Comment by PandaTsui** [comment]
Me too. The item had thrown a error same writer when I updated webpack to 4.0 .

**Comment by AJhinChen** [comment]
> It appears the .babelrc configuration only allows development to use import, this might need a fix or a good explanation of why that is perhaps.
> Fact is on master if you run npm run build, it will crash.

you are right. add the "plugins": ["dynamic-import-node"] to the production of .babelrc. it's work now.
